test/e2e-common.sh

```diff
@@ -235,11 +235,6 @@ data:
   profiling.enable: "true"
 EOF
 
-  echo ">> Adding more activator pods."
```

Comment:
Should we have a proper patch -min=2 -max=2 instead?

Reply:
`min=2 max=2` is the expected test case? Actually I sent this PR because I found that `503` error happened during the scaling down of Activator pods. (I have left the comment here https://github.com/knative/serving/issues/5781#issuecomment-541460365).

Reply:
We should at least bump the `min` amount via a patch in our tests, so we actually run with more than one activator. I'd leave `max` alone though, as fixing the amount of activator pods would hide issues as you've seen @nak3 .

Reply:
I see. I updated the `min` scale to `2` for testing. Thank you.